src/Compilers/CSharp/Portable/Errors/MessageProvider.cs

```diff
@@ -57,7 +57,7 @@ public override string GetHelpLink(int code)
             return ErrorFacts.GetHelpLink((ErrorCode)code);
         }
 
-        public override string GetCategory(int code)
+        public override LocalizableString GetCategory(int code)
         {
             return ErrorFacts.GetCategory((ErrorCode)code);
```

Comment:
Ah. I wondered about that, but not enough to dissuade myself from blithe optimism. Thanks, Manish, I'll do the rest of the plumbing.